TicketUtil.py

```diff
@@ -119,6 +118,123 @@ def close_requests_session(self):
         """
         self.s.close()
 
+class BugzillaTicket(Ticket):
+    """
+    A BZ Ticket object. Contains BZ-specific methods for working with tickets.
+    """
+
+    def __init__(self, base_url, project_key, ticket_id=None, auth=None, user=None, password=None):
+        self.ticketing_tool = 'Bugzilla'
+
+        # Right now, hardcode auth as 'kerberos', which is the only supported auth for BZ.
+        self.auth = auth if auth else 'kerberos'
+
+        self.credentials = {"login": user, "password": password}
+        self.token = None
+
+        # BZ URLs
+        self.base_url = base_url
+        self.rest_url = '{0}/rest/bug'.format(self.base_url)
+        self.auth_url = '{0}/rest/login'.format(self.base_url)
+
+        # Call our parent class's init method which creates our requests session.
+        super(BugzillaTicket, self).__init__(project_key, ticket_id)
+
+
+    def create_ticket_parameters(self, options_dict):
+
+        params = {"product": str(self.project_key)}
+
+        # Iterate through our options and add them to the params dict.
+        for key, value in options_dict.items():
+            params.update({str(key): str(value)})
+        return params
+
+    def create_requests_session(self):
+        if self.auth != 'rest':
+            return super()
+        else:
```

Comment:
No need for the `else` block here, since you're calling `return `one line above. If the `if` block is invoked the method is not executing the lines 156-172. Thanks to this you can avoid using the `else` -> one level less of indentation.